### PR TITLE
Refactor srf02 driver class. Move member variable initialization to declarations, standardize against other drivers and format.

### PR DESCRIPTION
**Pull request by mcsauder**
**Describe problem solved by the proposed pull request**
This PR migrates srf02 class member variable initialization to respective declarations, formats whitespace, and alphabetizes/organizes variable and method declarations in `srf02.cpp`.

This PR is copy/paste of the variable initialization values and method declarastion along with whitespace formatting.

**Describe your preferred solution**
Standardizing all of the distance sensor driver variable initialization, method ordering and general style will allow for future inheritance structure work to be performed on the distance sensor driver classes.

**Describe possible alternatives**
All of the distance sensor driver work could be accomplished in one massive PR, but breaking up work in each driver should minimize risk and reduce review effort.

**Additional context**
See #9279, #11853, #11857, #1858, #11859.

Please let me know if you have any questions on this PR. Thanks!

-Mark

### CODE REVIEW

**Comment by mcsauder** [comment]
@philipoe , would you be willing to review this PR for me?  I am trying to standardize the distance sensor drivers so that an inheritance hierarchy can be created.  Thank you!

**Comment by mcsauder** [comment]
Bench testing of this PR measuring a fixed distance object of approximately 25cm gives similar readings to PX4 master branch.

This PR:
image

PX4 master:
image

File: src/drivers/distance_sensor/srf02/srf02.cpp [hunk 1]
```diff
@@ -37,55 +37,52 @@
...
 
-#include <px4_config.h>
-#include <px4_defines.h>
-#include <px4_getopt.h>
-#include <px4_workqueue.h>
-#include <containers/Array.hpp>
-
-#include <drivers/device/i2c.h>
-
-#include <sys/types.h>
```
Comment (dagar) [inline src/drivers/distance_sensor/srf02/srf02.cpp]: Many of these includes probably aren't even needed.

Comment (mcsauder) [reply src/drivers/distance_sensor/srf02/srf02.cpp]: I pushed up a commit to delete the includes that are not needed. Thanks!

**Comment by mcsauder** [comment]
Hi @dagar, here are flight logs and plots of this PR and current master.  Performance is indistinguishable to me.

 Flight log and plots running the SRF02 on current master in position mode varying altitude between approximately 20cm and 2.5m: https://review.px4.io/plot_app?log=246e025c-b744-4176-9bf0-cbf545d1a303
image


Flight log and plots running the SRF02 from this PR in position mode varying altitude between approximately 20cm and 3.5m: https://review.px4.io/plot_app?log=99cb01c3-62b6-4999-bdba-f7bde750b010

image

Please let me know if you'd like to see anything else in this PR.  Thanks!

**Comment by mcsauder** [comment]
@dagar , ping.

**Comment by mcsauder** [comment]
If anyone in the world can prove that this PR has issues, I would like to speak with them about my experience.s :) Is there anyone out there with hardware to test this beside me?

**Comment by mcsauder** [comment]
@philipoe , @okalachev @julianoes , please weigh in on this PR. Thank you!

**Comment by mcsauder** [comment]
@davids5 and @dagar , The PX4:master stack size is insufficient for this driver.  Here is output from current PX4:master stackcheck when starting the driver:
image

I have added a commit to increase the stack size, here it output from loading a stackcheck build with this PR:
image

**Comment by mcsauder** [comment]
@davids5  and @dagar, there wasn't enough time to get to this in the dev call today. Would you mind letting me know what else you'd like done? Thanks!

**Pull request merged**